tests/a_scripts/inference/test_convert_composer_to_hf.py

```diff
@@ -1110,6 +1110,79 @@ def test_huggingface_conversion_callback(
     delete_transformers_cache()
 
 
+@patch('os.cpu_count', MagicMock(return_value=1))
+@patch(
+    'llmfoundry.callbacks.hf_checkpointer.SpawnProcess',
+    new=MockSpawnProcess,
+)
+def test_transform_model_pre_registration():
+
+    class ExtendedHuggingFaceCheckpointer(HuggingFaceCheckpointer):
+
+        def transform_model_pre_registration(self, model: PreTrainedModel):
+            self.using_peft = False
+            return super().transform_model_pre_registration(model)
+
+    model_cfg, tokenizer_name = _get_model_and_tokenizer(
+        model='llama2',
```

Comment:
switch to 'neo', otherwise the test will get skipped in CI due to not having an hf token